<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Starting point</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Difficulity</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DifficulitycomboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Slow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Medium</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fast</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="startComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Rautatieasema A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rosendahl</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sorinaukio</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="rulesButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rules</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
